### PR TITLE
Fix/5238 admin page interface improvements on mobile

### PR DESCRIPTION
**Pull request by gbonaspetti**
#5238 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
~- [ ] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)~ See #3030
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: frontend_app_admin_workspace_user/src/css/index.styl [hunk 1]
```diff
@@ -120,14 +120,21 @@
           color inherit
         & > i
           margin-right textSpacing
+        & .profileNavigation:hover
+          text-decoration underline
     &__active
       width 17%
     &__profile
-      width 62px // 62px it's the size of avatar's profile
+      width 49px // 49px it's the size of avatar's profile
```
Comment (Haskaris) [inline frontend_app_admin_workspace_user/src/css/index.styl:128]: Can't we use a global css variable for the size of avatar's profile? Just like `bigIconSize` below?

**Review by Haskaris — APPROVED** [review]

File: frontend_app_admin_workspace_user/src/css/index.styl (hunk 1 above)
Comment (gbonaspetti) [reply frontend_app_admin_workspace_user/src/css/index.styl:128]: will fix

Comment (gbonaspetti) [reply frontend_app_admin_workspace_user/src/css/index.styl]: fixed

**Comment by knodelle** [comment]
tested on branch

**Pull request merged**